packages/groqd/README.md

```diff
@@ -21,7 +21,7 @@ Read the official documentation at https://commerce.nearform.com/open-source/gro
 `GroqD` uses a strongly-typed chaining syntax to build queries:
 
 ```ts
-import { createGroqBuilder } from 'groqd';
+import { createGroqBuilder, zod } from 'groqd';
```

Comment:
🤔 Would it be better to just drop this, and do:
```
import { z } from 'zod'
```

Reply:
My only hesitation to refactor this to `import from 'zod'` is the fact that the `q.default(...)` utility is still relevant.  So I'm not going to make that sweeping change just yet.

I'll update these imports to `z`.